### DISCUSSION TITLE
[Album Import] Support to import albums from folder structure

### DISCUSSION THREAD

**Discussion by luis15pt** [immich-app/immich#10769] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The feature is already built and works, its provided in https://github.com/Salvoxia/immich-folder-album-creator

"This is a python script designed to automatically create albums in [Immich](https://immich.app/) from a folder structure mounted into the Immich container. This is useful for automatically creating and populating albums for external libraries. Using the provided docker image, the script can simply be added to the Immich compose stack and run along the rest of Immich's containers."

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4279.

Please search for duplicates before opening a new FR, as requested in the template
